client/lib/feed-stream-store/index.js

```diff
@@ -21,15 +20,15 @@ function feedKeyMaker( post ) {
 	return {
 		feedId: post.feed_ID,
 		postId: post.ID,
-		localMoment: moment( post.date ),
+		date: new Date( post.date ),
```

Comment:
i checked for things using localMoment and didn't find much. i altered those i did find.